IyiKuyoro/rmw-react-frontend
# #161198159 Landing page

#### What does this PR do?
Adds the setup for web pack and a basic react application

#### Description of Task to be completed?
> - Add the landing page components
> - Style all landing page elements
> - Make the landing page responsive for mobile devices

#### How can this be manually tested
To test the additions;
1. Clone the repo to your local machine
2. Checkout to the `ft-landing-page-#161198159` branch
3. Run `npm run dev` the browser should lunch the application on `PORT 8080`

#### Screenshots
![image](https://user-images.githubusercontent.com/30635021/47258483-be224c80-d493-11e8-8e5f-82b4ef648f26.png)
![image](https://user-images.githubusercontent.com/30635021/47258487-c7abb480-d493-11e8-8d4e-f590bd8f8e25.png)
![image](https://user-images.githubusercontent.com/30635021/47258495-d2664980-d493-11e8-9e54-3cde88d10ce9.png)
![image](https://user-images.githubusercontent.com/30635021/47258498-d6926700-d493-11e8-9e78-4cc75f94673d.png)

#### What are the relevant Pivotal Tracker stores?
#161198159
